db/patches/V1_6_61_05__draft_history.sql

```diff
@@ -0,0 +1,9 @@
+-- Keep a log of draft picks
+CREATE TABLE IF NOT EXISTS `draft_history` (
+  `draft_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `game_id` int(10) unsigned NOT NULL,
+  `leader_account_id` smallint(6) unsigned NOT NULL,
+  `picked_account_id` smallint(6) unsigned NOT NULL,
+  `time` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`draft_id`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8 AUTO_INCREMENT=1;
```

Comment:
It seems like there's a lot of discussion about which to use, but every time I see someone say "Just use X" there's always someone who says "Only do that if you need Y, otherwise use Z".

People often link [this article](https://www.percona.com/blog/2009/01/12/should-you-move-from-myisam-to-innodb/), and what I take away most from it is:
> Innodb Needs Tuning As a final note about MyISAM to Innodb migration I should mention about Innodb tuning. Innodb needs tuning. Really. MyISAM for many applications can work well with defaults.

In particular, I don't think we've done the level of analysis that is needed to tune the performance of whatever we're using to make it better than the alternative.

I'm happy with whatever you decide, but since it seems like a wash, I'll simply add that MyISAM is already tested. :)

Reply:
I do wonder if the issues brought up in that article about innodb have improved in the 8 years since, at the same time any performance consideration is unlikely to matter for our purposes

Reply:
Yeah, me too. I searched around for a while, but couldn't find anything that looked definitive.